app/views/layouts/application.html.erb

```diff
@@ -29,12 +29,9 @@
         </ul>
       </div>
       </nav>
-      <%# flash.each do |name, message| %>
-      <%#= content_tag :div, message, id: "flash_#{name}" %>
-      <%# end %>
-      <div>
-        <%= flash[:remove_item].html_safe unless flash[:remove_item].blank? %>
```

Comment:
do we still need some of this code so that we can see the link in the flash message?